.github/workflows/lint.yaml

```diff
@@ -49,7 +49,6 @@ jobs:
           working_directory: "tools/analyzers"
       - name: "Run custom analyzers"
         run: './tools/analyzers/analyzers -skip-pkg "github.com/authzed/spicedb/pkg/proto/dispatch/v1" -disallowed-nil-return-type-paths "*github.com/authzed/spicedb/pkg/proto/dispatch/v1.DispatchCheckResponse,*github.com/authzed/spicedb/pkg/proto/dispatch/v1.DispatchExpandResponse,*github.com/authzed/spicedb/pkg/proto/dispatch/v1.DispatchLookupResponse" ./...'
-      - uses: "authzed/actions/govulncheck@main"
```

Comment:
Why is this removed?

Reply:
govulncheck was broken in `latest`. They don't have any releases, so I removed it. It's fixed now so I can revert this